src/forwarder.spec.ts

```diff
@@ -285,4 +291,44 @@ describe("addCloudWatchForwarderSubscriptions", () => {
       }
     `);
   });
+  it("throws DatadogForwarderNotFoundError", async () => {
```

Comment:
I would update the name of the test to say WHEN/WHY it throws DatadogForwarderNotFoundError. I.e. "it throws DatadogForwarderNotFoundError when ..."